src/neptune/handler.py

```diff
@@ -730,7 +730,11 @@ def track_files(self, path: str, *, destination: str = None, wait: bool = False)
             attr.track_files(path=path, destination=destination, wait=wait)
 
     def __delitem__(self, path) -> None:
-        self.pop(path)
+        raise NeptuneUnsupportedFunctionalityException()
+
+    @check_protected_paths
+    def pop(self, path: str = None, *, wait: bool = False) -> None:
```

Comment:
@normandy7 We're missing docstrings here. Would you mind to add it to your "backlog" and try to add it in a separate PR? https://docs.neptune.ai/api/client_index/#pop That's on metadata level like:
```python
run["parameters/learninggg_rata"].pop()
```

Reply:
Sure, I will add it to my backlog that actually exists ;)